src/components/CheckBox.tsx

```diff
@@ -0,0 +1,6 @@
+export function CheckBox({ label, ...props }) {
+    return <label className="flex items-center gap-2 rounded-lg bg-white/10 hover:bg-white/20 py-1 px-2">
+        <input type="checkbox" className="appearance-none outline-none size-5 bg-black/30 rounded border border-solid border-custom-grey focus:ring-2 focus:ring-custom-purple focus:ring-offset-2 focus:ring-offset-custom-grey checked:border-none checked:bg-custom-purple checked:bg-tick-mark" {...props} />
```

Comment:
I belive, this needs `cursor:pointer` on hover

Reply:
Yes, missed to add it